rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt

```diff
@@ -960,7 +960,14 @@ data class CallStmt(
                     )
                 }
             }
-            targetProgramParams[index].name to interpreter[it.result.name]
+            /*
+             * TODO: On RPGLE is possible to have nested field with an access like this: ds.field.subfield.
+             *  Consider to refactor this logic when is necessary to add subfield capability.
+             */
+            val resultName = if (it.result.name.contains("."))
```

Comment:
Remove the TODO and replace it with: If there are two dots, we should throw an error. This makes it clear that this feature is not supported and helps avoid subtle errors.